tests/unit/resources/hint_sniffing/delimited-tsv-with-header.json

```diff
@@ -0,0 +1,13 @@
+{
+  "required": {
+    "header-row": true,
+    "escape": null,
+    "quoting": "minimal",
+    "field-delimiter": "\t"
+  },
+  "initial_hints": {
+    "quoting": "minimal",
+    "field-delimiter": "\t"
+  },
+  "notes": "TODO Lack of delimiter detection is painful, as tabs are hard to specify on the command line"
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker